pkgs/os-specific/linux/rtl8812au/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "rtl8812au-${kernel.version}-${version}";
-  version = "5.2.20.2_28373.20190903";
+  version = "5.6.4.2_35491.20200318";
 
   src = fetchFromGitHub {
-    owner = "zebulon2";
-    repo = "rtl8812au-driver-5.2.20";
-    rev = "30d47a0a3f43ccb19e8fd59fe93d74a955147bf2";
-    sha256 = "1fy0f8ihxd0i5kr8gmky8v8xl0ns6bhxfdn64c97c5irzdvg37sr";
+    owner = "gordboy";
+    repo = "rtl8812au-5.6.4.2";
+    rev = "49e98ff9bfdbe2ddce843808713de383132002e0";
+    sha256 = "0f4isqasm9rli5v6a7xpphyh509wdxs1zcfvgdsnyhnv8amhqxgs";
   };
```

Comment:
why has the upstream changed?

Reply:
The `zebulon2` repo is a fork of the `gordboy` one and in the past contained some additional fixes. Now the situation has reversed and the `gordboy` repo contains the most recent fixes for linux 5.6.